targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -137,21 +146,25 @@ public void OnApplicationFocus(bool isFocused)
         /// </summary>
         public void Send()
         {
-            if (PlayFabClientAPI.IsClientLoggedIn())
+            if ((PlayFabClientAPI.IsClientLoggedIn()) && (isSending == false))
             {
-                int eventsInTheBatch = 0;
+                isSending = true;
```

Comment:
If multi-threading is an issue, consider making isSending static.

Reply:
Unity is inherently single threaded. There's no real benefit of going out of our way to take this out of the Unity main-thread, so it's unlikely that this will ever have to deal with multi-thread issues.